phpreport-report

```diff
@@ -340,8 +340,15 @@ class DetailedReport(Report):
             self.formatter.generate_large_text("Onsite hours worked: %s" % DateUtils.format_delta(onsite_time))
 
     def get_stories_for_day_and_user(self, user, date):
+        def story_text(task):
```

Comment:
Maybe if the task doesn't have a story it doesn't make sense to add the time either? Also, if there is only one task for the day, isn't adding the story time a little redundant?